### PR TITLE
Added support for parsing a process description containing multiple processes

### PR DESCRIPTION
**Pull request by huard**
It's not clear to me how to test this. Should I create an xml file for this ?

### CODE REVIEW

**Comment by cehbrecht** [comment]
... a follow up of my comment. We get a list of process descriptions *only* when using `identifier=all`. If we always return a list in `describeprocess` we brake the interface. But maybe we can move the code for the distinction here:
https://github.com/geopython/OWSLib/blob/dcbc8db9a8eafa715a0469317bf23ea07de17326/owslib/wps.py#L291

In case of `identifier=all` it will *always* return a list (even empty or with a single process) ... otherwise a single process.

**Comment by cehbrecht** [comment]
Check an example for `describeprocess`:
https://github.com/geopython/OWSLib/blob/master/tests/test_wps_describeprocess_usgs.py

You can use a prepared xml document ... for example one coming from Emu.

**Comment by huard** [comment]
The failures look unrelated.

File: owslib/wps.py [hunk 1]
```diff
@@ -348,20 +348,27 @@ def _parseProcessMetadata(self, rootElement):
...
+                if p.identifier == process.identifier:
+                    self.processes[n] = process
+                    found = True
+            # otherwise add it
+            if not found:
+                self.processes.append(process)
+
+            processes.append(process)
+
+        if len(processes) == 1:
```
[the remark this answers is not in the document]
Comment (cehbrecht) [reply owslib/wps.py]: done.

**Comment by cehbrecht** [comment]
test failures are not related.

@huard If you are happy with this PR, I will merge.

**Comment by huard** [comment]
Good for me.

**Comment by cehbrecht** [comment]
@huard done. Thanks :)

**Pull request merged**